tests/unit/test_table.py

```diff
@@ -2436,6 +2479,61 @@ def test_to_dataframe_iterable_w_bqstorage(self):
         # Don't close the client if it was passed in.
         bqstorage_client._transport.grpc_channel.close.assert_not_called()
 
+    @unittest.skipIf(pandas is None, "Requires `pandas`")
+    @unittest.skipIf(
+        bigquery_storage is None, "Requires `google-cloud-bigquery-storage`"
+    )
+    @unittest.skipIf(pyarrow is None, "Requires `pyarrow`")
```

Comment:
This is what I was talking about, i.e. "semantic conflicts". We should be careful to not let these through, since they do not necessarily create loud merge conflicts.

It can get tricky :/

Reply:
Good catch!